adafruit_register/i2c_bcd_datetime.py

```diff
@@ -77,20 +77,25 @@ def __init__(self, register_address, weekday_first=True, weekday_start=1):
         else:
             self.weekday_offset = 1
         self.weekday_start = weekday_start
+        # Masking value list  n/a   sec   min   hour  days  wkday  mon  year
+        self.mask_datetime = [0xFF, 0x7F, 0x7F, 0x3F, 0x3F, 0x07, 0x1F, 0xFF]
```

Comment:
```suggestion
        self.mask_datetime = b"\xFF\x7F\x07F\x3F\x3F\x07\x1F\xFF"
```

Using a byte string will save space because it uses 1 byte per element rather than 4.

Reply:
@tannewt I think there's an extra zero in the third element? It should look like this, right?
```
self.mask_datetime = b"\xFF\x7F\x07F\x3F\x3F\x07\x1F\xFF"
```

I figured it would be helpful to keep it mutable that way the RTC class for a specific chip can alter it if it needs to grab additional register bits.

Reply:
Yup! I just did the edit on github. Please check my work and test it. :-)